src/js/profile/indicator.js

```diff
@@ -20,7 +21,12 @@ export class Indicator extends Component {
 
         isLast = _isLast;
 
-        this.addIndicatorChart(wrapper, title, indicatorData, detail);
+        if (indicatorData.chartConfiguration.rawText) {
```

Comment:
Could you explain rawText? What are you expecting to come from the API, plain, html, or markdown

Reply:
I was expecting html, renamed the variable to be more clear